charts/kube-prometheus-stack/values.yaml

```diff
@@ -1490,7 +1490,7 @@ kube-state-metrics:
   ##
   # image:
   #  repository: registry.k8s.io/kube-state-metrics/kube-state-metrics
-  #  tag: v2.5.0
```

Comment:
This should be done in a separate pull request for kube-prometheus-stack. It will also require a version bump.